### PR TITLE
Take home

### PR DESCRIPTION
**Pull request by segovia-95**
#2 "finished landing page"

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -46,38 +45,68 @@ <h1 class="display-4"></h1>
     </div>
     </div>
 
-<div class="d-flex justify-content-center">
-<h1>Main Entree's</h1>
+<div class="d-flex container">
+<div class="center-block">
+  <h2>Main Entree's</h2>
```
Comment (xyeras) [inline index.html:50]: **Main Entree's** looks like it is located within the jumbotron on my screen.
Screenshot (5)

File: index.html [hunk 2]
```diff
@@ -46,38 +45,68 @@ <h1 class="display-4"></h1>
...
+      <h2>Sopes</h2>
   <p>Sopes chicos con arroz mexicano.</p>
   </div>
   </div>
  </div>
 
+ <!--Get Your Ticket Today-->
+ <div class="d-flex container btn-vertical">
+  <h1>Get Your Grand Opening Reservation Today!</h1>
+  <p>Each Reservation includes complimentary grand opening surprise desert.</p>
```
Comment (xyeras) [inline index.html:84]: The reservation part doesn't look balanced on the same level as the other objects.
Screenshot (11)

**Review by xyeras — APPROVED** [review]
This is great! You got your grid and flex down really well. The styling is beautiful. 👍 I think the reason it looks weird on my part because we have devices with different screen sizes. I wasn't sure if the website looked normal on your screen versus mine. Haha. Great job, again! 😄

File: index.html (hunk 1 above)
Comment (segovia-95) [reply index.html:50]: Thank you!! And yes I am trying to see what can be done to move Main Entree's down and fix the reservation part to align with the other objects. Thank you again for your feedback Liz!

**Pull request merged**